tempesta_fw/http.c

```diff
@@ -2088,6 +2154,11 @@ static void
 tfw_http_cli_error_resp_and_log(bool reply, bool nolog, TfwHttpReq *req,
 				int status, const char *msg)
 {
+	if (!req->conn) {
+		tfw_http_msg_free((TfwHttpMsg *)req);
+		return;
+	}
```

Comment:
Later it'll be unclear what requests without a connection are, so please write a comment, or, better since there are  a few such places, write a small function for the requests deletion.

Reply:
Corrected. Replaced these pointer checks with `TFW_HTTP_HMONITOR` request flag verification, according to https://github.com/tempesta-tech/tempesta/pull/877#discussion_r160961463.